README.md

```diff
@@ -351,9 +351,8 @@ remote_syslog2 is written in go, and uses [govendor] to manage
 dependencies. To get everything set up, [install go][goinstall] then
 run:
 
-    go get github.com/kardianos/govendor
-    go get github.com/mitchellh/gox
-    go get github.com/papertrail/remote_syslog2
+    go install github.com/mitchellh/gox@latest
+    git clone git@github.com:papertrail/remote_syslog2.git
```

Comment:
https://golang.org/doc/go-get-install-deprecation
https://github.com/golang/go/issues/31529#issuecomment-484920157 discusses using `git clone` instead of `go get` for local development